pages/reference.js

```diff
@@ -21,12 +25,31 @@ import morse_styles from '../styles/morse.module.scss'
 import braille_styles from '../styles/braille.module.scss'
 import semaphore_styles from '../styles/semaphore.module.scss'
 import * as React from 'react'
+import { useAppDispatch, useAppSelector } from '../app/hooks'
+import * as slctr from '../features/reference/referenceSelector'
+import { favoriteStarClick } from '../features/reference/referenceSlice'
 
 export default function WordsPage() {
+  const dispatch = useAppDispatch()
+  const favorites = useAppSelector(slctr.getFavorites)
+
   const chars = AlphabetEn.split('')
-  const tableRowsJsw = chars.map((char, index) => {
+  const getCharRowJsx = (char) => {
+    const index = AlphabetEn.indexOf(char)
+    const starJsx = (
+      <TableCell>
+        <Button onClick={() => dispatch(favoriteStarClick({ char }))}>
+          {favorites.includes(char) ? (
+            <StarOutlinedIcon />
```

Comment:
Myslím, že zadávání pomocí klávesnice by bylo ve většině případů rychlejší než pomocí klikání na hvězdičky. U současného řešení je navíc rušivé, že po každém kluknutí ti odskočí řádek, na který jsi klikl. Zadávání pomocí klávesnice by výhledově usnadnilo zadávání jiných znaků než 26 písmen (problematičtější by bylo "ch")